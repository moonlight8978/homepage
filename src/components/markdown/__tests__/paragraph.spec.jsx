import React from 'react'
import { render } from '@testing-library/react'

import Paragraph from '../paragraph'

test('renders string', () => {
  const { container } = render(<Paragraph>Hello</Paragraph>)
  expect(container.firstChild).toMatchSnapshot()
})

test('renders react node', () => {
  const { container } = render(
    <Paragraph>
      <span>Hello</span>
    </Paragraph>
  )
  expect(container.firstChild).toMatchSnapshot()
})
