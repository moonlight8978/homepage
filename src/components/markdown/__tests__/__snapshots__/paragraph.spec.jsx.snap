// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders react node 1`] = `
<div
  class="paragraph"
>
  <span>
    Hello
  </span>
</div>
`;

exports[`renders string 1`] = `
<div
  class="paragraph"
>
  Hello
</div>
`;
